blocks/identity-block/components/social-sign-on/_children/GoogleSignIn.jsx

```diff
@@ -1,5 +1,54 @@
-import React from "react";
+import React, { useEffect } from "react";
 
-const GoogleSignIn = () => <div id="google-sign-in-button" />;
+import { Button, Icon, usePhrases } from "@wpmedia/arc-themes-components";
+
+import { SIGN_UP } from "../constants";
+
+const GoogleIcon = <Icon name="GoogleColor" />;
+
+function GoogleSignIn({ customButtons, socialSignOnIn, className }) {
+	const phrases = usePhrases();
+	const googleNotification = phrases.t("identity-block.social-google-one-tap-notification");
+
+	useEffect(() => {
+		if (customButtons) {
+			document.getElementById("custom-google-signin-btn").addEventListener("click", () =>
+				window.google.accounts.id.prompt((notification) => {
+					if (notification.isSkippedMoment()) {
+						// https://developers.google.com/identity/gsi/web/reference/js-reference
+						// https://developers.google.com/identity/gsi/web/guides/features#exponential_cooldown
+						// eslint-disable-next-line
+						alert(googleNotification);
+						// Remove cookie works in Safari
+						document.cookie = `g_state=;path=/;expires=Thu, 01 Jan 1970 00:00:01 GMT`;
+						window.google.accounts.id.prompt();
+					}
+				}),
+			);
```

Comment:
Again, the same thing here.  Why are we not just adding this to the onClick for the button?